README.md

```diff
@@ -1,5 +1,181 @@
+
+Our code base is powered by the [Robot Operating System](https://www.ros.org) (ROS), and written primarily in Python.
+
+The following components make up our software stack:
+
+- Onboard:
+    * [Controls](onboard/catkin_ws/src/controls) - Determines thruster outputs given a current and desired state.
+    * [Data Pub](onboard/catkin_ws/src/data_pub) - Collects and parses data from sensors and publishes it for use by other packages.
+    * [Offboard Comms](onboard/catkin_ws/src/offboard_comms) - Allows communication between ROS code and the onboard Arduino.
+    * [Static Transforms](onboard/catkin_ws/src/static_transforms) - Makes static transforms available for other packages.
+    * [Computer Vision](onboard/catkin_ws/src/cv) - Locates objects (goals/obstacles) via camera input and machine learning.
+    * [Execute](onboard/catkin_ws/src/execute) - Houses launch files that simplify starting and stopping our stack.
+    * [Motion Planning](onboard/catkin_ws/src/motion_planning) - Determines and defines the tasks and motion of the robot.
+    * [Sensor Fusion](onboard/catkin_ws/src/sensor_fusion) - Interprets sensor data and publishes an estimation of the current robot state.
+- Landside:
+    * [Joystick](landside/catkin_ws/src/joystick) - Allows manual joystick control for testing.
+    * GUI - User interface for monitoring the status of the robot during testing.
+    * [Simulation](simulation) - Physics-enabled simulation that can be used for local testing.
+    * [Camera View](landside/catkin_ws/src/camera_view) - Package that allows for viewing, saving, and loading videos to simulate camera input.
+
+
+## Flow 
+
+The general flow of information between components is shown in the diagram below:
+
+(Diagram to be added)
+
+
+## Running Our Docker Images
+
+### Required Software
+1. Download and install the appropriate Docker client.
+
+    * [Mac](https://docs.docker.com/docker-for-mac/install/)
+    * [Windows (Home)](https://docs.docker.com/docker-for-windows/install-windows-home/)
+    * [Windows (Pro, Education, etc.)](https://docs.docker.com/docker-for-windows/install/)
+
+    :information_source: Which Windows do I have? *Settings > System > About > look at "Edition"*
+
+2. If you want graphics forwarding over SSH:
+
+    * [Mac: XQuartz](https://www.xquartz.org/)
+    * [Windows: MobaXterm](https://mobaxterm.mobatek.net/)<br>
+        :warning: Open MobaXterm > click on Settings > go the X11 tab > uncheck RANDR. This allows us to use [rviz](http://wiki.ros.org/rviz) over SSH.
+
+:information_source: The password for the `root` user in both containers is `robotics`.
```

Comment:
Moved to be next to each ssh command so people don't miss it, despite the fact that it's repeated. (It may have to be repeated in the future anyway if we have different passwords for each container.)